web/netdata-switch-dashboard.sh.in

```diff
@@ -28,12 +28,12 @@ switch_dashboard() {
 
     for dashboard in classic react ; do
         # shellcheck disable=SC2013
-        for path in $(cat ${webdir}-${dashboard}/.files) ; do
-            rm -f "${webdir}/${path}"
+        for item in $(cat ${webdir}-${dashboard}/.files) ; do
```

Comment:
```suggestion
        for file in $(cat ${webdir}-${dashboard}/.files) ; do
```

Reply:
I tend to preferentially avoid naming variables `file` in shell scripts because that happens to be the name of a command found on many Linux systems and some BSD systems.